memory/notes/CombatManager.md

```diff
@@ -0,0 +1,101 @@
+
+- TODO: Character HP
+- TODO: Boost (AP) on ground
+- Cutscenes
```

Comment:
There's also the state of the demo boss (ie the physical/magical modes it can switch between) for vulnerabilities and so on.

Reply:
Yep!

So on the demo boss, theres actually a bunch of unique classes for the fight, so i'm expecting to see this more.

![image](https://github.com/shenef/SoS-TAS/assets/5025835/399e4d59-f30f-4d5b-9ebd-3f2047eeb3c6)